source/_docs/composer.md

```diff
@@ -1,175 +1,87 @@
+
+This is possible on Pantheon by specifying `web_docroot: true` in `pantheon.yml` file. For details, see [Serving Sites from the Web Subdirectory](/docs/nested-docroot/).
+
+## Source and Artifact Workflow
+<div class="flex-panel-group">
+  <div class="flex-panel-item">
+    <div class="flex-panel-body">
+      <div class="flex-panel-title">
+        <h4 class="info" style="margin-top:10px;font-size:larger">GitHub</h3>
+        <div class="pantheon-official">
+          <img alt="GitHub Logo" src="/source/docs/assets/images/github-logo.svg" class="main-topic-info__plugin-image" style="max-width:40px;margin-bottom:10px!important;">
+          <p class="pantheon-official"></p>
+        </div>
+      </div>
+      <p class="topic-info__description"><a href="https://github.org">GitHub</a> is an online service that provides cloud storage Git repositories that may be cloned and used locally, or edited directly through their web-based management interface. These features are very useful to teams collaborating on a project together.</p>
+    </div>
+  </div>
+  <div class="flex-panel-item">
+    <div class="flex-panel-body">
+      <div class="flex-panel-title">
+        <h4 class="info" style="margin-top:10px;font-size:larger">CircleCI</h3>
+        <div class="pantheon-official">
+          <img alt="CircleCI Logo" src="/source/docs/assets/images/circleci-logo.svg" class="main-topic-info__plugin-image" style="max-width:40px;margin-bottom:10px!important;">
+          <p class="pantheon-official"></p>
+        </div>
+      </div>
+      <p class="topic-info__description"><a href="https://circleci.com">CircleCI</a> provides hosted services to run automated tests for a project, and GitHub provides an integration to run these tests to whenever a change is submitted. The process of testing each set of changed files prior to merging them into the main branch is called continuous integration.</p>
+    </div>
+  </div>
+</div>
 
-We recommend managing custom code as plugins or modules within individual repositories and hosting each one remotely on a service such as [GitHub](https://github.com/), [Bitbucket](https://bitbucket.org), or [GitLab](https://about.gitlab.com/). You can then use Composer to manage and organize the software needed for your project to run, including custom plugins or modules.
+Only files unique to the project are tracked as part of the project's main "source" repository on GitHub, which requires an abstraction layer to compile dependencies and deploy an entire "artifact" to the site repository on Pantheon. The abstraction layer is facilitated by CirlceCI in the Pantheon maintained examples, but the principles are the same for other continuous integration service providers.
```

Comment:
Is this too technical? I tried to wordsmith it but got stuck

Reply:
I could't possibly think of a simpler way to phrase that 👍

Reply:
This sounds great to me! @alexfornuto I think I accidently deleted your comment while trying to move mine, sorry about that!

Reply:
That's OK @dustinleblanc, I pulled it out of #docs:

> I couldn't possibly think of a simpler way to phrase that :+1: